db/schema.rb

```diff
@@ -291,11 +291,13 @@
     t.float    "application_location_lat"
     t.float    "application_location_lng"
     t.text     "application_giving_back"
-    t.text     "application_diversity"
     t.string   "confirmation_token"
     t.datetime "confirmed_at"
     t.datetime "confirmation_sent_at"
     t.string   "unconfirmed_email"
+    t.text     "application_diversity"
+    t.string   "Expertise"
+    t.string   "Interests"
```

Comment:
I feel like it's no big deal, but I'd prefer for this to be consistent (so: no capitals). @Anagha22 could you change this, please?